drivers/modem/hl7800.c

```diff
@@ -1707,16 +1726,22 @@ static void dns_work_cb(struct k_work *work)
 #if defined(CONFIG_DNS_RESOLVER) && !defined(CONFIG_DNS_SERVER_IP_ADDRESSES)
 	int ret;
 	struct dns_resolve_context *dnsCtx;
-	static const char * const dns_servers_str[] = { ictx.dns_string, NULL };
+	static const char *const dns_servers_str[] = { ictx.dns_v4_string,
```

Comment:
I remembered that open braces should be the last char in a line?
```c
static const char *const dns_servers_str[] = {
        ictx.dns_v4_string,
        ...
```

Reply:
This file was auto-formatted with the .clang-format file using VSCode. So I am going to leave this as is.